rfc/php_technical_committee.wiki

```diff
@@ -52,83 +51,121 @@ For example if there is a proposal to add accessors for PHP classes, then everyt
+    @-mentioning the TC GitHub team.
+  - An issue with the `tc-hold` tag MUST NOT be merged.
+  - The TC members will confer in whatever manner they deem most convenient, public or private or both.
+    * The TC SHOULD consult with relevant other individuals, including the core developers involved in the issue, Release
+      Managers, and impacted extension maintainers as appropriate.
+  - The TC MUST deliver a decision within one month, via the mechanism described below.
+  - The TC Secretary MUST announce the decision and reasoning for it on the issue in question and remove the `tc-hold` tag.
+  - The decision of the TC is binding.  That means a relevant change MUST NOT be merged until and unless it conforms
+    with the TC's decision.  Depending on the decision, the core developer involved may merge the issue, close it, or
+    revise it accordingly.
 
-Any member can decide to drop out of being part of the TC. The TC can continue in reduced number unless their number
-goes below 3. If that happens, the new election needs to be scheduled within a month. If the number of member is 3 or
-more but the secretary is the member that dropped out, then s new secretary must be selected.
+If an issue has been marked `tc-hold` for more than two weeks with no response or acknowledgement from the TC, the
+developer SHOULD post to the Internals mailing list a notification of absence.
+
+If a notification of absence receives no response from the TC or any member of it for one month, then the entire TC
+is considered AWOL.  The senior-most Release Manager of the current stable release will step in to act as TC Secretary
+and immediately call for a new vacancy election as described above.
+
+The election of a new TC will "reset the clock" on any outstanding `tc-hold` issues.
 
 ==== Decision process ====
 
-=== RFC ===
+=== User-facing changes ===
+
+The RFC process will continue to be used for user facing changes.  However, it will no longer be used for
+non-user-facing technical decisions.
+
+TC members have no special role or authority in the RFC process.
+
+=== Process changes ===
+
+The RFC process will continue to be used for process changes, including changes to the role of the TC.
+
+=== Technical changes ===
+
+As noted, the TC has no special role or authority on user-facing changes.  The TC may, however, decide on the
+implementation of those features. In other words the TC can decide about implementation and ABI but not about API.
 
-The RFC process will continue to be used for user facing changes. It will also continue to be used for all process
-changes. However, it will no longer be used for any technical decisions.
+The TC SHOULD NOT block the merging of a user-facing change approved by the RFC process, unless the provided implementation
+would result in security issues or significant performance penalties.  If so, TC members are expected to actively participate
```

Comment:
I understand the reasoning here but I'm not sure we need this as badly implemented change might potentially create other issues with project maintenance or possibly the developer might not respect comments of other core devs during review so it should be really decided on case by case basis. It might make sense to just put there something like that TC should always try to find a solution for getting the implementation merged in some form that fulfils the accepted RFC.

Reply:
I mostly included this section to call out that this isn't intended to create an "end run" around the RFC process; if an RFC passes, but two of the TC members voted no and really hate it, they still can't use that position to block its acceptance.

This is largely political, in other words.  I'm open to suggestions on how to reword it.

Reply:
The intention is good here. I think we could just make it a bit broader so maybe something like
```suggestion
would result in introduction of new bugs, side effects not mentioned in the RFC, significant performance penalties or if there is a different implementation in progress that TC finds more appropriate.
```
What do you think? Feel free to change the wording / extend ofc.

Reply:
Tweaked very slightly from what you list.